include/crow/app.h

```diff
@@ -156,8 +156,9 @@ namespace crow
         /// Run the server on multiple threads using a specific number
         self_t& concurrency(std::uint16_t concurrency)
         {
-            if (concurrency < 1)
-                concurrency = 1;
+            if (concurrency < 2) // using 2 instead of 1 because 1 will be subtracted later.
+                concurrency = 2;
+            concurrency--; // Account for the main (acceptor) thread creating all the other threads.
```

Comment:
Maybe assign `concurrency_ = concurrency - MAIN_THREAD_COUNT;`? Or maybe some better name ;)

Reply:
Well there's only 1 main thread, so it doesn't really make sense to do that..

Reply:
Yeah, but then assigning a number to subtract from it next line is also a bit of nonsense ;)